Source/swiftlint/Commands/Lint.swift

```diff
@@ -19,6 +19,8 @@ extension SwiftLint {
         var noCache = false
         @Flag(help: "Run all rules, even opt-in and disabled ones, ignoring `only_rules`.")
         var enableAllRules = false
+        @Option(help: "Run only the specified rule, ignoring the configuration file.")
```

Comment:
It's not that the configuration file is ignored completely, is it? Only the rules specified in there are overridden, yet configurations for the "only rule" still apply. Perhaps this should be made clearer here.

Reply:
So I went with: "Run only the specified rule, ignoring `only_rules`, `optin_rules` and `disabled_rules`.", by analogy with `--enable-all-rules` "Run all rules, even opt-in and disabled ones, ignoring `only_rules`."

Reply:
Sounds good. It's ~~`optin_rules`~~`opt_in_rules` though.

Reply:
🤦